pkg/frontend/kubeactions/kubeactions_test.go

```diff
@@ -135,7 +129,10 @@ func TestFindGVR(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			ka := &kubeactions{}
 
-			got := ka.findGVR(tt.resources, tt.kind, "")
+			got, err := ka.findGVR(tt.resources, tt.kind, "")
+			if err != nil && err.Error() != tt.wantError {
```

Comment:
```suggestion
			if err != nil && err.Error() != tt.wantError ||
				err == nil && tt.wantError != "" {
```

getting in the habit of writing ^ is better because it also covers the case that an error was expected but none was returned.